HomeTask3 v2.0/HomeTask3 v2.0/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace HomeTask3_v2._0
+{
+    class Program
+    {
+        static void Main()
+        {
+         Console.WriteLine($"Значение x для варианта 16 равно {Method16(2, 3) + Method16(3, 7) + Method16(5, 5)}");
```

Comment:
При выводе на консоль следует представить значение с точностью до трех знаков после запятой.
```suggestion
         Console.WriteLine($"Значение x для варианта 16 равно {Method16(2, 3) + Method16(3, 7) + Method16(5, 5), F3}");
```